### PR TITLE
Formatting wallet README files as Markdown.

### PR DESCRIPTION
**Pull request by jonathancross**
Hi @schildbach This is the first of 3 files being converted per #389
I recommend viewing diffs with the Markdown preview option  :-)

### CODE REVIEW

**Comment by schildbach** [comment]
Do you plan to convert the other two README's too? I'd rather convert all three in one go.

**Comment by jonathancross** [comment]
I was hoping to use feedback on this to inform the next 2, but sure, I can add the others to this PR.

**Comment by schildbach** [comment]
I briefly looked over it and it seemed fine.

**Comment by jonathancross** [comment]
Cool, ready for review.  :-)

**Comment by schildbach** [comment]
Thanks. Hmm, this PR does much more that just format as markdown. Please don't change any wording, except if the change is mandated by the reformat to keep the READMEs consistent. Independent wording changes should go into different commits at least.

File: wallet/README.md [hunk 1]
```diff
@@ -85,25 +81,26 @@ use
...
+To install the app on your Android device, use:
 
     # first time only
     sudo apt install android-tools-adb
 
     # each time
     adb install wallet/build/outputs/apk/bitcoin-wallet-debug.apk
 
-If installing fails, make sure "Developer options" and "USB debugging" are enabled on your Android device, and an ADB
+If installation fails, make sure "Developer options" and "USB debugging" are enabled on your Android device, and an ADB
```
Comment (schildbach) [inline wallet/README.md]: Example for an unrelated wording change.

Comment (jonathancross) [inline wallet/README.md]: Done.

**Comment by jonathancross** [comment]
I went through and removed many non-critical changes in words, white-space, punctuation, etc.  
Diffs should be much easier to read now. :-)

**Comment by schildbach** [comment]
Thanks a lot, merged. Looks very nice now!

If you want, you can rebase your previous branch against current master which should result in a commit with all the content changes.

**Comment by jonathancross** [comment]
Thanks @schildbach 
I'll send over a final PR with the word / punctuation (and maybe white-space) changes when I get a chance.
